conf/logback.xml

```diff
@@ -9,7 +9,7 @@
     </appender>
 
     <appender name="FILE" class="ch.qos.logback.core.FileAppender">
-        <file>${application.home}/logs/donut.log</file>
```

Comment:
Not sure if we should push this change to the main branch instead of trying to find the root of the problem. You can always change your own version of logback.xml until then.